modules/con_notifications-configuration.adoc

```diff
@@ -98,23 +98,23 @@ spec:
 apiVersion: argoproj.io/v1alpha1
 kind: NotificationsConfiguration
 metadata:
- name: default-notifications-configuration # <1>
+ name: default-notifications-configuration #<1>
 spec:
- subscriptions: |
-  subscriptions: | # <2>
+ subscriptions: #<2>
+  subscriptions: |
     # subscription for on-sync-status-unknown trigger notifications
     - recipients:
-    - slack:test2
-    - email:test@gmail.com
-    triggers:
-    - on-sync-status-unknown
+      - slack:test2
+      - email:test@gmail.com
+      triggers:
+      - on-sync-status-unknown
     # subscription restricted to applications with matching labels only
     - recipients:
-    - slack:test3
-    selector: test=true
-    triggers:
-    - on-sync-status-unknown
-icon: <override-icon> # optional icon for the message (supports both emoji and url notation)
+      - slack:test3
+      selector: test=true
+      triggers:
+      - on-sync-status-unknown
+    icon: <override-icon> # optional icon for the message (supports both emoji and url notation)
```

Comment:
this line may be removed I think. If we are having it, I think the indendation is wrong.

Reply:
I got it, I have fixed it now. PTAL.